pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -189,8 +189,25 @@ func handleInit(ctx *verifierContext) {
 
 	log.Functionln("handleInit")
 
-	// Create VerifyImageStatus for objects that were verified before reboot
-	handleInitVerifiedObjects(ctx)
+	// Run a periodic timer so we always update StillRunning
+	stillRunning := time.NewTicker(25 * time.Second)
+	defer stillRunning.Stop()
+	waitForVerifiedObjectsChan := make(chan bool, 1)
+	go func() {
+		// Create VerifyImageStatus for objects that were verified before reboot
+		handleInitVerifiedObjects(ctx)
+		waitForVerifiedObjectsChan <- true
+	}()
+
+	objectsVerified := false
+	for !objectsVerified {
+		select {
```

Comment:
Would it make sense to add a log.Notice that we are waiting for the initial objects to be verified?

Reply:
Added, and also for the end of the verification